AssetRelationsViewer/Editor/VisualizationNode/VisualizationNode.cs

```diff
@@ -2,48 +2,59 @@
 using Com.Innogames.Core.Frontend.NodeDependencyLookup;
 using UnityEditor;
 using UnityEngine;
+using UnityEngine.Profiling;
 
 namespace Com.Innogames.Core.Frontend.AssetRelationsViewer
 {
-    public class VisualizationNode : VisualizationNodeBase
+	public class VisualizationNode : VisualizationNodeBase
 	{
+		public ITypeHandler TypeHandler;
 		public VisualizationNodeData NodeData;
 		public string Key;
 		public int Hash;
 
+		public bool HasNonFilteredChildren = false;
+		public bool Filtered;
+
 		public void SetKey(string value)
 		{
 			Key = value;
 			Hash = value.GetHashCode();
 		}
 
-		public override string GetSortingKey(RelationType relationType)
+		public override string GetSortingKey(RelationType relationType, bool sortBySize)
```

Comment:
Do you always sort by string? if sorting by size you could compare the number values directly